Makefile

```diff
@@ -19,3 +19,9 @@ local:
 
 start_all:
```

Comment:
I will remove `start_all` since `start_all_tmux` replaces that. As far as the way make targets are defined, they are defined in a consistent way across all services which makes it easy to loop through all services and start them.